examples/cstg/html/index.html

```diff
@@ -75,8 +75,14 @@
     </script>
   </head>
   <body>
-    <h1>Server-Side Integration Example, UID2 JavaScript SDK</h1>
-    <!-- TODO: Add introductory text here. -->
+    <h1>Client-Side Integration Example, UID2 JavaScript SDK</h1>
+    <p>
+      This example demonstrates how a content publisher can follow the 
+      <a href="https://unifiedid.com/docs/guides/publisher-client-side"
+      >Client-Side Integration Guide for JavaScript </a
```

Comment:
nit: can we please put the closing tag on the same line as the opening? i.e.
```
  >Client-Side Integration Guide for JavaScript
</a>
```

Reply:
ok i think i fixed as you requested?

Reply:
close, line 82 should be indented:
```
<a href="https://unifiedid.com/docs/guides/publisher-client-side"
  >Client-Side Integration Guide for JavaScript
</a>
```

Reply:
Auto-formatter in your IDE should hopefully fix this automatically?

Reply:
fixed as discussed offline